src/App.tsx

```diff
@@ -1,26 +1,135 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { USER_ID, deleteTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import classNames from 'classnames';
+import { Filter } from './types/EnumFilter';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Error } from './types/EnumError';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<Filter[keyof Filter]>(Filter.all);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [todosAreLoadingIds, setTodosAreLoadingIds] = useState<number[]>([]);
```

Comment:
Thank you for your advice) But I can't now understand, how to create variable, because I use this state to turn on loader (sometimes it is only one todo which I'm deleting, sometimes - all completed todos, like you show). And I'm going to use this state to turn on loader while toggling, so this state will keep sometimes only completed todos, sometimes only active. So will it be all right to use this state as it is?